dev_guide/builds/build_inputs.adoc

```diff
@@ -731,86 +731,115 @@ archive, and, if valid, the builder changes into that subdirectory before
 
 . Create the secret, if it does not exist:
 +
 ----
-$ oc create secret generic secret-npmrc \
-    --from-file=.npmrc=<path/to/.npmrc>
+$ oc create secret generic secret-mvn \
+    --from-file=id_rsa=<path/to/.ssh/id_rsa>
 ----
 +
-This creates a new secret named *_secret-npmrc_*, which contains the base64
-encoded content of the *_~/.npmrc_* file.
+This creates a new secret named *_secret-mvn_*, which contains the base64
+encoded content of the *_id_rsa_* private key.
 
-. Add the secret to the `source` section in the existing `BuildConfig`:
+. Add the configMap and secret to the `source` section in the existing 
+`BuildConfig`:
 +
 [source,yaml]
 ----
 source:
   git:
-    uri: https://github.com/openshift/nodejs-ex.git
+    uri: https://github.com/wildfly/quickstart.git
+  contextDir: helloworld
+  configMaps:
+    - configMap:
+        name: settings-mvn
   secrets:
     - secret:
-        name: secret-npmrc
+        name: secret-mvn
 ----
 
-To include the secret in a new `BuildConfig`, run the following command:
+To include the secret and configMap in a new `BuildConfig`, run the following 
+command:
 
 ----
 $ oc new-build \
-    openshift/nodejs-010-centos7~https://github.com/openshift/nodejs-ex.git \
-    --build-secret secret-npmrc
+    openshift/wildfly-101-centos7~https://github.com/wildfly/quickstart.git \
+    --contextDir helloworld --build-secret “secret-mvn” \
+    --build-configMap "settings-mvn"
 ----
 
-During the build, the *_.npmrc_* file is copied into the directory where the
-source code is located. In {product-title} S2I builder images, this
-is the image working directory, which is set using the `WORKDIR` instruction
-in the *_Dockerfile_*. If you want to specify another directory, add a
-`destinationDir` to the secret definition:
+During the build, the *_settings.xml_* and *_id_rsa_* files are copied into the
+directory where the source code is located. In {product-title} S2I builder 
+images, this is the image working directory, which is set using the `WORKDIR` 
+instruction in the *_Dockerfile_*. If you want to specify another directory, 
+add a `destinationDir` to the secret definition:
 
 [source,yaml]
 ----
 source:
   git:
-    uri: https://github.com/openshift/nodejs-ex.git
+    uri: https://github.com/wildfly/quickstart.git
+  contextDir: helloworld
+  configMaps:
+    - configMap:
+        name: settings-mvn
+      destinationDir: ".m2"
   secrets:
     - secret:
-        name: secret-npmrc
-      destinationDir: /etc
+        name: secret-mvn
+      destinationDir: ".ssh"
 ----
 
 You can also specify the destination directory when creating a new
 `BuildConfig`:
 
 ----
 $ oc new-build \
-    openshift/nodejs-010-centos7~https://github.com/openshift/nodejs-ex.git \
-    --build-secret “secret-npmrc:/etc”
+    openshift/wildfly-101-centos7~https://github.com/wildfly/quickstart.git \
+    --contextDir helloworld --build-secret “secret-mvn:.ssh” \
+    --build-configMap "settings-mvn:.m2"
```

Comment:
looks like it's `--build-config-map`? 
https://github.com/openshift/origin/pull/19655/files#diff-924f9baa301d1772b396b1629eb2bb15R118